patchwork/api/patch.py

```diff
@@ -373,6 +375,26 @@ def get_queryset(self):
         )
 
 
+class PatchDetailPermission(PatchworkPermission):
+    non_delegate_editable_fields = set(['planning_to_review'])
+
+    def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
+            return True
+
+        data = request.data
+
+        if 'planning_to_review' in data:
```

Comment:
You need to *only* this field is included, right?

```suggestion
        if list(data) == ['attention_set']:
```

I'd like to see a test for this also.